platform/darwin/README.md

```diff
@@ -0,0 +1,7 @@
+# Darwin
+
+The code in the Darwin platform targets Apple platforms but is not specific
+to iOS or OS X. This code is not distributed as an SDK in itself, but is required
+by the [iOS](https://github.com/mapbox/mapbox-gl-native/tree/master/platform/ios)
```

Comment:
Nit: When practical, avoid using “the iOS SDK” or “the OS X SDK” to refer to a Mapbox SDK. “The” iOS or OS X SDK is what Apple publishes.